packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -144,12 +145,26 @@ async function getTransitiveDependencies(packageName, externals) {
 }
 
 async function copyJsModule(from, to) {
-    await copy(from, to, {
-        filter: async path =>
-            (await promises.lstat(path)).isDirectory()
-                ? !/^(android|ios|windows|macos|\.?(github|gradle)|__(tests|mocks)__|docs|jest|examples?)$/.test(
-                      basename(path)
-                  )
-                : /.*\.(jsx?|json|tsx?)$/.test(extname(path)) || basename(path).toLowerCase().includes("license")
-    });
+    return new Promise((resolve, reject) =>
```

Comment:
🎄  You can use [`promisify`](https://nodejs.org/api/util.html#util_util_promisify_original) to avoid this manual `new Promise` hassle

Reply:
Fixed